lib/Models/WebMapServiceCatalogGroup.ts

```diff
@@ -288,7 +288,18 @@ class GetCapabilitiesStratum extends LoadableStratum(
     if (!isDefined(this.catalogGroup.uniqueId)) {
       return;
     }
-    return `${this.catalogGroup.uniqueId}/${layer.Title}`;
+    return `${this.catalogGroup.uniqueId}/${layer.Name ?? layer.Title}`;
+  }
+
+  /** For backward-compatibility.
+   * If layer.Name is defined, we will use it to create layer autoID (see `this.getLayerId`).
+   * Previously we used layer.Title, so we now add it as a shareKey
+   */
+  getLayerShareKeys(layer: CapabilitiesLayer) {
+    if (isDefined(layer.Name) && layer.Title !== layer.Name)
+      return [`${this.catalogGroup.uniqueId}/${layer.Title}`];
+
+    return [];
```

Comment:
These changes should keep most v7 IDS working

https://github.com/TerriaJS/terriajs/blob/6153d332aa3bd160ddbbd968be37a339a4f6ad96/lib/Models/WebMapServiceCatalogGroup.js#L263-L271

Reply:
I can always add a shareKey for `layer.Abstract` to support all values of `titleField`

Reply:
Actually I just found this

https://github.com/TerriaJS/terriajs/blob/5d13c2ba093fa0e35aeaaf27178622c4a1034625/architecture/0006-dynamic-groups-and-shareKeys.md#L106-L108

Maybe I should just do this?